interfaces/seccomp/template.go

```diff
@@ -218,8 +218,8 @@ fadvise64
 fadvise64_64
 arm_fadvise64_64
 
-mbarrier
 mbind
+membarrier
```

Comment:
I'm not sure what this test would look like because some syscalls are architecture dependent. The best we could do would be to take the list we have in http://bazaar.launchpad.net/~snappy-dev/ubuntu-core-launcher/trunk/view/head:/debian/README.syscalls and make sure that the syscalls in the seccomp policy are in that list. Note, the list will of course need to be updated from time to time. Also a typoed syscall is not fatal to the system as a whole because of the way the launcher sets the seccomp sandbox (obviously the actual syscall is not available though).